src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,16 @@
-export const TodoList = () => {};
+import { TodoWithUser } from '../../App.types';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: TodoWithUser[];
+};
+
+export const TodoList = ({ todos }: Props) => {
```

Comment:
I mentioned `return-right-away` approach somewhere above, this component is a great place to practice it, since we have 2 return statement that can be avoided here